tpx3awkward/examples/example.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
It's great to have a few examples like this, however committing output cells of a notebook to a git repo is generally considered bad practice. We already have a tool, `nbstripout`, for cleaning up the notebooks before committing them. It can be activated as a pre-commit hook. Before committing, simply run `pre-commit run --all` (or pick a specific file) -- it will do some formatting, linting, and make sure the the code conforms to the stylistic standards.